projects/ndpi/build.sh

```diff
@@ -62,13 +66,17 @@ make install
 
 cd /nDPI
 
-./autogen.sh
+LDFLAGS="-lpcap" ./autogen.sh --with-only-libndpi
```

Comment:
We'd better remove build directory, autogen may contain some cache. rm -rf build_dir may be better than make clean

Reply:
Every run of autogen.sh call autoreconf - re-creating configure and makefiles.